doc/antora/modules/reference/pages/raddb/mods-available/yubikey.adoc

```diff
@@ -59,11 +59,11 @@ Yubikey authentication needs two attributes retrieved from persistent storage:
 [options="header,autowidth"]
 |===
 | Attributes                                         | Description
-| `&control.Vendor-Specific.Yubicon.Yubikey-Key`     | The AES key used to decrypt the OTP data.
+| `control.Vendor-Specific.Yubicon.Yubikey-Key`     | The AES key used to decrypt the OTP data.
```

Comment:
to note, all this is automatically generated, you need to update the config files in `raddb/mods-available` otherwise all your changes will be reverted.